src/Symfony/Component/Serializer/Mapping/AttributeMetadata.php

```diff
@@ -114,6 +123,14 @@ public function setSerializedName(string $serializedName = null)
         $this->serializedName = $serializedName;
```

Comment:
Shouldn't this method be deprecated, and call the new `addSerializedName()` one?

Reply:
done but I let the serializedName assignation as it is a public property that can be used ?

Reply:
The property is marked as internal, it is public only for performance and shouldn't be accessed directly. It isn't covered by our BC promise.

Reply:
done, but should I also delete completely the property serializedName ? it is still used in merge method for AttributeMetadata and CompiledClassMetadataFactory

Reply:
If we can remove it, let's do it.

Reply:
done, you can review